sdk/textanalytics/Azure.AI.TextAnalytics/src/AspectSentiment.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
```

Comment:
Should this be in `.Models`?  Ditto for the other models.

Reply:
hmm very interesting. I didn't do it because all of the main objects have been in the main namespace. But now that you mention it, this is a slightly more advanced scenario so it is worth moving into the Models namespace.
I will do that

Reply:
Oh shoot - you didn't have a `.Models` namespace at all yet!  (The api listings are helpful but things like that don't pop out as clearly.)  Hmmm.  I guess I wouldn't add a new namespace for just these three and we should think about this again the next time we add a bunch of new models in one big feature release.